src/Core/Binding/ServerConnection.cs

```diff
@@ -29,25 +29,34 @@ public abstract class ServerConnection
     public ICredentials Credentials { get; set; }
     
     public abstract Uri ServerUri { get; }
+    public abstract Uri CredentialsUri { get; }
 
     private ServerConnection(string id, ServerConnectionSettings settings = null, ICredentials credentials = null)
     {
         Id = id ?? throw new ArgumentNullException(nameof(id));
         Settings = settings ?? DefaultSettings;
         Credentials = credentials;
     }
-    
-    public sealed class SonarCloud(string organizationKey, ServerConnectionSettings settings = null, ICredentials credentials = null)
-        : ServerConnection(organizationKey, settings, credentials)
+
+    public sealed class SonarCloud : ServerConnection
     {
-        public string OrganizationKey { get; } = organizationKey;
+        public const string Organizations = "organizations";
```

Comment:
not sure if it was necessary to create a constant just for one usage

Reply:
I am also referencing it in the tests. I think it is better to be a constant. If we ever update this, than it will also be updated in the tests

Reply:
Should it be automatically updated in the tests? I get the argument for things like UI strings, but this feels like a value that should only change intentionally and we should strictly prohibit any unintentional changes to URIs.

Reply:
I don't think anybody is changing "unintentionally" constants in code. But sure, I can make it hard-coded instead (although hard-coded strings are usually against clean code principles)